### DISCUSSION TITLE
[Help] Black artefacts when converting gif to images.

### DISCUSSION THREAD

**Discussion by Nisanez** [ImageMagick/ImageMagick#2007] [General] [answered]
Hi!

I am trying to convert a gif to a list of images, but I am getting these black lines. 

The command I am using is `convert -coalesce animated-train-image-0019.gif ./imagesFolder/image_%05d.png`

the gif I am trying to convert is https://www.animatedimages.org/data/media/75/animated-train-image-0019.gif

The results I am getting are:
![image_00000](https://user-images.githubusercontent.com/16883711/81701261-6f237a00-9472-11ea-89fa-5d5e200fb2a2.png)
![image_00001](https://user-images.githubusercontent.com/16883711/81701266-6fbc1080-9472-11ea-8171-dfa3063549fc.png)
![image_00002](https://user-images.githubusercontent.com/16883711/81701269-7054a700-9472-11ea-90f6-a706face525e.png)
![image_00003](https://user-images.githubusercontent.com/16883711/81701276-7185d400-9472-11ea-8600-350aec091a25.png)

I understand there is a transparency optimisation, but I wasn't manage to get pass that.

Any idea how can I do that?

**Comment by dlemstra** [comment]
And the version that you are using is?

**Reply by Nisanez** [reply]
Thank you for replying!
The version I have is `Version: ImageMagick 7.0.8-58 Q16 x86_64 2019-08-01`

**Comment by dlemstra** [comment, marked as answer]
It seems that you found a bug. A patch to fix this has just been pushed to the master. You will get the correct images in the next release (`7.0.10-12`).

**Reply by Nisanez** [reply]
Great! thank you
